spec/support/spree/config_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module ConfigHelper
+    def with_spree_config(hash)
+      stored = config_slice(hash.keys)
+      Config.set(hash)
+      yield
+      Config.set(stored)
```

Comment:
I just wanted to make sure the following is true:

> My understanding is that .set just changes the value of the settings corresponding to the keys provided.

If instead all the other config keys are empty we have a problem